addon/-private/data-view/radar/radar.js

```diff
@@ -215,7 +226,16 @@ export default class Radar {
     });
   }
 
+  updateItems() {
```

Comment:
This is a real pain, but basically we only want to do a `reset/prepend/append` when the items have changed, but we don't want to run or cache the final indexes _until_ the RAF because the array can change in the interim (especially in the case of PromiseArrays)